DevAcademy/Environment/Coordinator.swift

```diff
@@ -13,11 +13,11 @@ final class Coordinator: ObservableObject{
         PlacesScene()
         
     }
-    func PlacesDetailScene(with feature: Feature) -> some View{
-        PlacesDetail(state: PlacesDetailStateView(feature: feature))
+    func placesDetailScene(with feature: Feature) -> some View{
```

Comment:
Jenom takovej kosmetickej detail, ve Swiftu je zvykem mit pred slozenou zavorkou vzdy mezeru, takze idealne by to melo vypadat nejak takhle:
```Swift
func placesDetailScene(with feature: Feature) -> some View {
```

Vsiml jsem si toho na vic mistech, tak to prosim fixni vsude. 😉